resilience4j-all/src/main/java/io/github/resilience4j/decorators/Decorators.java

```diff
@@ -382,7 +384,7 @@ private DecorateCheckedSupplier(CheckedFunction0<T> supplier) {
         }
 
         public DecorateCheckedSupplier<T> withRateLimiter(RateLimiter rateLimiter, int permits) {
-            supplier = RateLimiter.decorateCheckedSupplier(rateLimiter, permits, supplier);
+            supplier = VavrRateLimiter.decorateCheckedSupplier(rateLimiter, permits, supplier);
```

Comment:
@RobWin I have currently added dependency `resilience4j-vavr` to `resilience4j-all` module. Do you think we should create `VavrDecorators` inside `resilience4j-vavr` module and move all vavr related decorators in that class and remove dependency `resilience4j-vavr` from `resilience4j-all` module.

Reply:
Yes, that makes sense

Reply:
sure. I will do it next PR.